scripts/eurostat/health_determinants/common/EuroStat.py

```diff
@@ -0,0 +1,359 @@
+sys.path.insert(1, _COMMON_PATH)
+# pylint: disable=wrong-import-position
+from common.replacement_functions import (_split_column, _replace_col_values)
+from common.sv_config import file_to_sv_mapping
+
+# For import util.alpha2_to_dcid
+_COMMON_PATH = os.path.abspath(
+    os.path.join(os.path.dirname(__file__), '../../../../'))
+sys.path.insert(1, _COMMON_PATH)
+
+from util.alpha2_to_dcid import COUNTRY_MAP
+# pylint: enable=wrong-import-position
+
+_FLAGS = flags.FLAGS
+default_input_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),
+                                  "input_files")
+flags.DEFINE_string("input_path", default_input_path, "Import Data File's List")
+
+_MCF_TEMPLATE = ("Node: dcid:{pv1}\n"
+                 "{pv11}\n"
+                 "typeOf: dcs:StatisticalVariable\n"
+                 "populationType: dcs:Person{pv2}{pv3}{pv4}{pv5}"
+                 "{pv6}{pv7}{pv8}{pv9}{pv10}\n"
+                 "statType: dcs:measuredValue\n"
+                 "measuredProperty: dcs:count\n")
+
+_TMCF_TEMPLATE = ("Node: E:eurostat_population_{import_name}->E0\n"
+                  "typeOf: dcs:StatVarObservation\n"
+                  "variableMeasured: C:eurostat_population_{import_name}->SV\n"
+                  "measurementMethod: C:eurostat_population_{import_name}->"
+                  "Measurement_Method\n"
+                  "observationAbout: C:eurostat_population_{import_name}->geo\n"
+                  "observationDate: C:eurostat_population_{import_name}->time\n"
+                  "scalingFactor: 100\n"
+                  "value: C:eurostat_population_{import_name}->observation\n")
+
+
+class EuroStat:
+    """
+    This Class has requried methods to generate Cleaned CSV,
+    MCF and TMCF Files.
+    """
+
+    def __init__(self, input_files: list, csv_file_path: str,
+                 mcf_file_path: str, tmcf_file_path: str,
+                 import_name: str) -> None:
+        self._input_files = input_files
+        self._cleaned_csv_file_path = csv_file_path
+        self._mcf_file_path = mcf_file_path
+        self._tmcf_file_path = tmcf_file_path
+        self._import_name = import_name
+
+    def _parse_file(self, file_name: str, df: pd.DataFrame,
+                    import_name: dict) -> pd.DataFrame:
+        split_columns = df.columns.values.tolist()[0]
+        df = _split_column(df, split_columns)
+
+        split_columns = split_columns.replace('isced97', 'isced11')\
+                .replace('geo\time','geo').replace('geo\\time','geo')\
+                .replace('time\geo','time').replace('time\\geo','time')\
+                .replace('quantile', 'quant_inc')
+
+        df.rename(columns={
+            'geo\time': 'geo',
+            'geo\\time': 'geo',
+            'time\geo': 'time',
+            'time\\geo': 'time',
+            'isced97': 'isced11',
+            'quantile': 'quant_inc'
+        },
+                  inplace=True)
```

Comment:
instead of two separate sets statements to rename list and actual columns in df here and below, we could rename columns in the df directly, then get the final list of split_columns from the df directly once.